src/constants.ts

```diff
@@ -15,14 +16,19 @@ export enum ItemValidationReviewStatuses {
   Pending = 'pending',
 }
 
-export const ItemValidationProcesses = {
-  BadWordsDetection: {
-    name: 'bad-words-detection',
-  },
-  AggressiveAndHateSpeech: {
-    name: 'aggressive-or-hate-speech-detection',
-  },
-  ImageChecking: {
-    name: 'image-checking',
-  },
-};
+export enum ItemValidationProcesses {
+  BadWordsDetection = 'bad-words-detection',
+  AggressiveAndHateSpeech = 'aggressive-langauge-classification',
+  ImageChecking = 'image-classification'
+}
+
+export enum ITEM_TYPE {
+  LINK = 'embeddedLink',
+  APP = 'app',
+  DOCUMENT = 'document',
+  FOLDER = 'folder',
+  S3FILE = 's3File',
+  LOCALFILE = 'file',
+}
+
+export const PREDICTION_THRESHOLD = 0.3;
```

Comment:
It is linked to a specific classifier right? Could you precise in the naming of the variable?